corona-warn-app/cwa-app-android
# Expand line in data sharing onboarding information layout (EXPOSUREAPP-7202) closes #3131

### Description

This PR fixes the issue "Truncated text output (Data Sharing information)" described in https://github.com/corona-warn-app/cwa-app-android/issues/3131.

The TextView  in `fragment_ppa_more_info.xml` responsible for displaying `onboarding_ppa_more_info_rki_data_point_5_text` was missing the attribute `android:layout_weight="1"`. Without this attribute, the TextView was vertically-limited to one line of text, which was insufficient to output the whole text `"Information about whether you received a positive or negative test result through the app."` on an averagely sized mobile phone.

### Steps to reproduce

#### Environment

Device name: Google Pixel 3a emulator
Android version: 11
Display font size and Display size both set to Default

#### Verification steps

1. Open CWA
2. Tap three-dot icon
3. Tap Settings > Share Data
4. Scroll down to bottom "Detailed Information about This Data Processing and Data Protection Risks in the U.S. and Other Third Countries" and tap on this string.
5. In "Detailed Information about Data Sharing" scroll down to section "To this end, the RKI collects the following data:" and further to sub-section "If you have retrieved a test result through the app:"
6. The next bullet point should be displayed as "Information about whether you received a positive or negative test result through the app.", not truncated, such as "Information about whether you received a pos"

![7202 fix](https://user-images.githubusercontent.com/66998419/117706875-21009e00-b1ce-11eb-8298-dd1020c39e6e.jpg)

### Acknowledgements

Thanks to @fynngodau for his helping hand!


---
Internal Tracking ID: [EXPOSUREAPP-7202](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7202)
